reformulator.py

```diff
@@ -323,13 +326,13 @@ def handle_exception(exc_type, exc_value, exc_traceback):
             anki(action="notesInfo", notes=nids)
         ).set_index("noteId")
         self.notes = self.notes.loc[nids]
-        assert not self.notes.empty, "Empty notes df"
+        assert not self.notes.empty, "Empty notes"
```

Comment:
Well it's not that the notes are empty but that no notes are found

Reply:
Just fixing a typo here

Reply:
(df standed for dataframe here as in pandas but it's not important I agree)